packages/studio/api/mail/index.ts

```diff
@@ -225,17 +264,20 @@ function getMailTemplateComponents(templateFilePath: string) {
 
   const components = []
 
-  // `export function X(){};`
   const exportedComponents = ast.body.filter((node: any) => {
-    return node.type === 'ExportDeclaration'
+    return [
+      'ExportDeclaration',
+      'VariableDeclaration',
+      'ExportDefaultDeclaration',
+      'ExportDefaultExpression',
```

Comment:
It looks like we only handle the `ExportDeclaration` and `ExportDefaultExpression` in the `extractParamAndValue` function. Including `VariableDeclaration` here also means that any non-exported variable is getting picked up and treated as a default export. 

I think we should remove those two types from here and only support them for just now without worrying about the `ExportDefaultDeclaration` as that is even more complex.

Reply:
Ah ok, thanks @Josh-Walker-GM , I'll have a look into that